circle.yml

```diff
@@ -1,33 +1,13 @@
 machine:
     services:
         - docker
-    python:
-        version: "2.7"
 
 checkout:
     post:
         - mkdir /home/ubuntu/catkin_ws
         - mkdir /home/ubuntu/catkin_ws/src
-        - mkdir /home/ubuntu/catkin_ws/src/pyrostester
 
-dependencies:
-    pre:
-        - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
-        - sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net --recv-key 0xB01FA116
-        - sudo apt-get update -qq
-        - sudo apt-get install -y python-catkin-pkg python-rosdep ros-indigo-catkin ros-indigo-ros ros-indigo-roslaunch build-essential
-        - pip install --upgrade pip
-    post:
-        - pip install -e /home/ubuntu/$CIRCLE_PROJECT_REPONAME
-        - cp -r test /home/ubuntu/catkin_ws/src/pyrostester
-        - pip install pytest
 
 test:
-    pre:
-        - source /opt/ros/indigo/setup.sh
-          #- rosdevel
     override:
-        - echo $PWD:
-            pwd: ../catkin_ws
-        - pytest src/pyrostester/test:
-            pwd: ../catkin_ws
+        - deactivate && ./test.sh
```

Comment:
Oh did you move most things to `test.sh`? Why is it run after `deactivate` though? Seems like you want to flip these?